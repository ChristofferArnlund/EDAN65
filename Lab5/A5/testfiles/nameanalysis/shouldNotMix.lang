//Should not mix localvars with functions
int m() {
	int x = 5;
	return x(); //should find x and complain about that
}
