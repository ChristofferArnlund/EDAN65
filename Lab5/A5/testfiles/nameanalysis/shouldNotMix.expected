Error at line 4: Function call 'x' does not point to function
